news/2019-01-09-introducing-yuzu.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Introducing Yuzu's New Look"
+date: 2019-01-09 09:00:00 +0000
+---
+
+Heeey! I'm back! You won't believe the sorta stuff I've been up to but wow, is it just me or am I *fashionable* now? Come check me out!
+
+![](https://assets.ppy.sh/media/yuzu/header.jpg)
+
+It's me, Yuzu! 
+
+Yeah, I'm pretty sure most of you were wondering what happened to me since you guys dropped in suggestions on my *refashionising* project (yeah, yeah, I know this isn't a real word, but I'm *wayyy* too cool to care.).
+
+Hold up a sec, lemme lock the doors first.
+
+Okay, where was I? Yeah, "refashionising". Ah, I love it when they scream in pain every time I use that word. Here, lemme flip this mirror around so you can see how gorgeous I am:
+
+![](https://assets.ppy.sh/media/yuzu/yuzu-full-cover.jpg)
+
+I picked through all of the things people suggested (a wizard hat? really?) decided to go with [Thievley](https://osu.ppy.sh/users/4717672)'s design because I felt you know... *connected* to her design. It screams... *speed*. 
+
+Okay, really, more like because she made me look pretty cool.
+
+I'm sorry that it took a *VERY* long time to talk about this, but here I am. Better late than never. A lot of things happened recently. Cool things. Really. Oh man, you won't believe the stuff that went down. It was ace.
+
+I choose to go with blue because I like blue. That's it. It's cool.
+
+Oh, oh, but lemme tell you how I got my amazing scarf!
+
+It starts with me accidentally dropping into a mysterious portal, which leads to some epic mecha fight (and flatten some medieval looking cities in the process, unfortunately), and the death of a friend that I recently met. 
+
+That was sad. But she had this really awesome scarf that she got from her grandfather - and now, it's mine! She gave it to me. I have proof. Really.
```

Comment:
maybe add an exclamation point after the last "really"
article reads fine since it's hard to make errors with this style